dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/TaskDefinitionController.java

```diff
@@ -160,14 +153,14 @@ public Result queryTaskDefinitionVersions(@ApiIgnore @RequestAttribute(value = C
         @ApiImplicitParam(name = "code", value = "TASK_DEFINITION_CODE", required = true, dataType = "Long", example = "1"),
         @ApiImplicitParam(name = "version", value = "VERSION", required = true, dataType = "Int", example = "100")
     })
-    @GetMapping(value = "/version/switch")
+    @GetMapping(value = "/{code}/version/switch/{version}")
```

Comment:
Get or Post ?

Get:
/{code}/version/{version}

Post:
/{code}/version/{version}/switch

Reply:
I think the first is good

Reply:
/{code}/version/{version}/switch better